### DISCUSSION TITLE
Where to submit a crash report? (running build 1.10.2)

### DISCUSSION THREAD

**Discussion by MartinParis** [sandboxie-plus/Sandboxie#3186] [General]
Should I create an issue and attach the dump file there?
![image](https://github.com/sandboxie-plus/Sandboxie/assets/7744829/e73c1c24-caa1-49f4-995f-44aa577710e4)

**Comment by DavidXanatos** [comment]
please first try if the issue is resolved with 1.10.4

**Comment by MartinParis** [comment]
Thanks, note to self, always check if there is a newer version :-).
From 1.10.2 to 1.10.4 install fails, cannot install because of something that is running, it tries three times, finishes the install and tells me at the end to install again.
That does not work like the first time, through upgrade from 1.10.4 itself, because it already is 1.10.4 (I guess).
Removed everything, including sandboxes, installing 10.4 with the exe from Git hub.
